### ISSUE TITLE
Issue with parser

### ISSUE THREAD

**Issue by asmoaesl** [asmoaesl/rsc#1]
```
>sqrt 3 / sqrt 5
Function(
    Sqrt,
    Constant(
        3.0
    )
)
1.7320508075688772
>2 + sqrt(3)
Constant(
    2.0
)
2
```

**Comment by asmoaesl** [comment]
It originates inside the tokenizer and only occurs when functions are involved.

**Comment by asmoaesl** [comment]
I haven't gotten around to fixing it because I've been so busy with school, but I'll give it a shot right now.

**Comment by asmoaesl** [comment]
3392b57163c1611744d554914b5c5341f0658172

**Comment by asmoaesl** [comment]
Solved with 3392b57163c1611744d554914b5c5341f0658172
